doc/dev/source/installation/installation_prod.rst

```diff
@@ -1,5 +1,8 @@
-Installation guide
-=================================================
+Installation guide (production)
+===============================
+
+.. todo::
+    Update instructions
```

Comment:
I'd actually remove the outdated instructions for now. Almost all of them are obsolete.